src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -121,19 +121,27 @@ class SyncControllerSpec extends FlatSpec with Matchers {
     peerMessageBus.reply(MessageFromPeer(NodeData(Seq(stateMptLeafWithAccount)), peer2.id))
     peerMessageBus.expectMsg(Unsubscribe(MessageClassifier(Set(NodeData.code), PeerSelector.WithId(peer2.id))))
 
+    //wait for peers throttle
+    Thread.sleep(2.second.toMillis)
+    //trigger scheduling
+    time.advance(2.second)
```

Comment:
If it's set to 100.millis for tests why are we waiting 2 seconds? Shouldn't we be using that config value here? Also doesn't the scheduler time already advance when we sleep?